run2auau/TrackingProduction/Fun4All_SingleJob0.C

```diff
@@ -0,0 +1,142 @@
+/*
+ * This macro is run in our daily CI and is intended as a minimum working
+ * example showing how to unpack the raw hits into the offline tracker hit
+ * format. No other reconstruction or analysis is performed
+ */
+#include <GlobalVariables.C>
+#include <QA.C>
+#include <Trkr_Clustering.C>
+#include <Trkr_LaserClustering.C>
+#include <Trkr_RecoInit.C>
+
+#include <fun4all/Fun4AllDstInputManager.h>
+#include <fun4all/Fun4AllDstOutputManager.h>
+#include <fun4all/Fun4AllInputManager.h>
+#include <fun4all/Fun4AllOutputManager.h>
+#include <fun4all/Fun4AllRunNodeInputManager.h>
+#include <fun4all/Fun4AllServer.h>
+#include <fun4all/Fun4AllUtils.h>
+
+#include <trackingqa/InttClusterQA.h>
+
+#include "inttrawhitqa/InttRawHitQA.h"
+#include <ffamodules/CDBInterface.h>
+#include <ffamodules/FlagHandler.h>
+#include <phool/recoConsts.h>
+
+#include <stdio.h>
+
+R__LOAD_LIBRARY(libfun4all.so)
+R__LOAD_LIBRARY(libffamodules.so)
+R__LOAD_LIBRARY(libintt.so)
+R__LOAD_LIBRARY(libinttrawhitqa.so)
+R__LOAD_LIBRARY(libtrackingqa.so)
+void Fun4All_SingleJob0(const int nEvents = 2,                                    //
+                        const int runnumber = 54280,                              //
+                        const std::string outfilename = "inttcluster_ppg02.root", //
+                        const std::string dbtag = "ProdA_2024",                   //
+                        const std::string filelist = "filelist.list"              //
+)
+{
+    bool saverawhit = true;
```

Comment:
Does this need to be true? Don't the raw hit DSTs that have already been produced hold these nodes, so we would be duplicating their information in the output DST from this macro?

Reply:
Yes there are rawhit DSTs already produced, but I was thinking to have all information in 1 DST in case Cheng-Wei and/or other PPG/INTT group members need rawhits for their studies. I could default it to false

Reply:
In that case then, you are fine for us to delete the initial raw hit DSTs?

Reply:
Sorry, I am confused now. If the initial raw hit DSTs are deleted, how would I be able to get the rawhit DST with this macro?

Reply:
Sorry I was not clear. I meant after the production of the cluster DSTs, if there is a single DST with clusters and raw hits, there is no need for the felix by felix raw hit DSTs. So then we could delete them

Reply:
I personally don’t think it’s a significant issue not saving the raw hit nodes in the output DST from this macro, as it follows the production convention (actually, I would prefer we follow the production convention). It's just some collaborators want it and I was trying to accommodate. I’ll go ahead and remove the flag, so the raw hit nodes won’t be saved in the output DST. If someone really needs the rawhit dst, they can always go to the initial ones